server/sonar-web/src/main/js/apps/component-measures/sidebar/DomainFacet.js

```diff
@@ -80,11 +80,17 @@ export default class DomainFacet extends React.PureComponent {
     const items = addMeasureCategories(domain.name, filterMeasures(domain.measures));
     const hasCategories = items.some(item => typeof item === 'string');
     const translateMetric = hasCategories ? getLocalizedCategoryMetricName : getLocalizedMetricName;
-    const sortedItems = sortMeasures(domain.name, items);
+    let sortedItems = sortMeasures(domain.name, items);
+
+    sortedItems = sortedItems.filter((item, index) => {
+      if (item !== 'string') return true;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add curly braces around the nested statement(s) in this "if" block. ([javascript:CurlyBraces](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3ACurlyBraces))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fpm%2Ffix-sonar-10452&issues=AWH60zaka5Ltb7kATuXC&open=AWH60zaka5Ltb7kATuXC)

[ISSUE_KEY]: AWH60zaka5Ltb7kATuXC